pkgs/by-name/ai/aider-chat/package.nix

```diff
@@ -0,0 +1,109 @@
+{
+  lib,
+  stdenv,
+  python311,
```

Comment:
Why not using `python3`? It's not compatible with 3.12?

Reply:
It's because of tensorflow. See https://github.com/NixOS/nixpkgs/pull/323927#issuecomment-2270345032

Reply:
Fair enough!